wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -918,6 +925,15 @@ instance IxSet.Indexable (AccountIndex ': SecondaryAccountIxs)
                          (OrdByPrimKey Account) where
     indices = ixList
 
+-- | Datatype wrapping addresses for per-field endpoint
+newtype AccountAddresses = AccountAddresses
+    { acaAddresses :: [WalletAddress]
```

Comment:
I might be inclined to scrap this extra type if it was added only for "aesthetic" purposes to make the API description reads nicer and not for a real need. One less type to maintain 😉